docs/ui/components/AskPageAI/AskPageAIChat.tsx

```diff
@@ -0,0 +1,440 @@
+            onClick={handleClose}>
+            <XIcon className="icon-sm text-palette-white" />
+          </Button>
+        </div>
+      </div>
+
+      <div className="min-h-0 flex-1 overflow-y-auto px-5 py-6">
+        {conversation.length > 0 ? (
+          <div className="space-y-5">
+            {conversation.map((qa, index) => {
+              const isLast = index === conversation.length - 1;
+              const key = (qa.id as ConversationKey) ?? `${qa.question}-${index}`;
+              const displayQuestion = askedQuestions[index] ?? qa.question;
+
+              return (
+                <div key={key} className="space-y-2">
+                  <div className="flex justify-end pr-1">
+                    <div className="ml-auto max-w-[85%] rounded-md border border-default bg-subtle px-3 py-1.5 text-right text-sm leading-snug text-secondary shadow-xs">
+                      {displayQuestion}
+                    </div>
+                  </div>
+                  <div className="rounded-md border border-default bg-subtle px-3 py-2 shadow-xs">
+                    <FOOTNOTE className="font-medium text-default">AI Assistant</FOOTNOTE>
+                    <div className="prose prose-sm mt-1 text-secondary dark:prose-invert prose-a:text-link prose-pre:!bg-transparent">
+                      {qa.answer ? (
+                        <ReactMarkdown remarkPlugins={[remarkGfm]} components={markdownComponents}>
+                          {qa.answer}
+                        </ReactMarkdown>
+                      ) : (
+                        <FOOTNOTE theme="secondary">
+                          {isLast && isBusy ? 'Preparing answer...' : ''}
+                        </FOOTNOTE>
+                      )}
+                    </div>
+                  </div>
+                  {qa.sources?.length ? (
+                    <FOOTNOTE theme="secondary" className="ml-1 text-xs">
+                      Sources:{' '}
+                      {qa.sources.map((source, sourceIdx) => (
+                        <span key={source.source_url}>
+                          <a
+                            className="text-link"
+                            href={source.source_url}
+                            onClick={event => {
+                              if (event.metaKey || event.ctrlKey || event.button !== 0) {
+                                return;
+                              }
+                              handleNavigation(event);
+                            }}>
+                            {source.title || `Source ${sourceIdx + 1}`}
+                          </a>
+                          {sourceIdx < qa.sources.length - 1 ? ', ' : ''}
+                        </span>
+                      ))}
+                    </FOOTNOTE>
+                  ) : null}
+                </div>
+              );
+            })}
+          </div>
+        ) : (
+          <FOOTNOTE theme="secondary">Ask a question about {contextLabel}.</FOOTNOTE>
```

Comment:
I think this should always stay visible, also the `{contextLabel}` should be highlighted (maybe bold?)

Reply:
In fact, maybe that can be part of he header? Or sticky at the top? Basically making it clear what the context is.